internal/cephfs/snapshot.go

```diff
@@ -137,65 +137,69 @@ func getSnapshotInfo(ctx context.Context, volOptions *volumeOptions, cr *util.Cr
 }
 
 func protectSnapshot(ctx context.Context, volOptions *volumeOptions, cr *util.Credentials, snapID, volID volumeID) error {
-	args := []string{
-		"fs",
-		"subvolume",
-		"snapshot",
-		"protect",
-		volOptions.FsName,
-		string(volID),
-		string(snapID),
-		"--group_name",
-		volOptions.SubvolumeGroup,
-		"-m", volOptions.Monitors,
-		"-c", util.CephConfigPath,
-		"-n", cephEntityClientPrefix + cr.ID,
-		"--keyfile=" + cr.KeyFile,
-	}
+	if !checkSubvolumeHasFeature("snapshot-autoprotect", volOptions.Features) {
```

Comment:
```
if checkSubvolumeHasFeature("snapshot-autoprotect", volOptions.Features) {
  return nil
}
```

Reply:
I didn't follow this approach with an idea to keep the `return` statements minimum. The current code also looks to be doing the same thing to me. If this is a better approach, I would love to know the reason :)

Reply:
with this approach, the number of changed lines of code is 3+3(which makes review easier) lines for protect and unprotect isn't it?

Reply:
> with this approach, the number of changed lines of code is 3+3(which makes review easier) lines for protect and unprotect isn't it?

Thanks for explaining :) I'll change it accordingly.